### PR TITLE
feature flag to show/hide 'Raise Hand' button

### PR DESCRIPTION
**Pull request by patidars**
- Added feature flag 'raise-hand.enabled' to enable/disable 'Raise Hand' feature for Android/iOS.

### CODE REVIEW

File: react/features/toolbox/components/native/RaiseHandButton.js [hunk 1]
```diff
@@ -97,16 +98,16 @@ class RaiseHandButton extends AbstractButton<Props, *> {
...
  * @param {Object} state - The Redux state.
  * @private
- * @returns {{
- *     _raisedHand: boolean
- * }}
+ * @returns {Props}
  */
 function _mapStateToProps(state): Object {
     const _localParticipant = getLocalParticipant(state);
+    const visible = getFeatureFlag(state, RAISE_HAND_ENABLED, true);
```
Comment (saghul) [inline react/features/toolbox/components/native/RaiseHandButton.js:105]: Make this the default, if visible wasn’t given in ownProps.

**Comment by patidars** [comment]
@saghul I did required changes. Please have a look.

**Comment by saghul** [comment]
@patidars Can you please fix the linter errors?

**Comment by patidars** [comment]
@saghul I fixed. Thanks

File: react/features/base/flags/constants.js [hunk 2]
```diff
@@ -69,6 +69,12 @@ export const MEETING_PASSWORD_ENABLED = 'meeting-password.enabled';
  */
 export const PIP_ENABLED = 'pip.enabled';
 
+/**
+ * Flag indicating if raise hand feature should be enabled.
+ * Default: auto-detected.
```
Comment (saghul) [inline react/features/base/flags/constants.js:74]:
```suggestion
 * Default: enabled.
```

**Comment by saghul** [comment]
Cheerss!

**Pull request merged**